### DISCUSSION TITLE
Page view charts on the sites page

### DISCUSSION THREAD

**Discussion by toniengelhardt** [plausible/analytics#2790] [Anything Else]
First of all, I love Plausible, thanks for making it!

There is only one thing that bugs me:

If you have multiple projects in Plausible it would make lot of sense to have the timeline chart for each of them on the `sites` page, aka. instead of this

<img width="1680" alt="Screenshot 2023-03-25 at 15 23 17" src="https://user-images.githubusercontent.com/2703233/227729277-797605bb-4698-4742-8b6d-1e3135fbe882.png">

only one project per row and this chart 

<img width="1115" alt="Screenshot 2023-03-25 at 16 21 35" src="https://user-images.githubusercontent.com/2703233/227729364-5fe24bda-21d5-4756-9bb0-41e111fc4a1b.png">

with 1/3 the height and without the numbers for each of them.

On top should be a selector for timeframe and which of the metrics each of the charts shows.

When I open Plausible every day (and i think it is similar for a lot of devs), what I usually want to see is the page view chart for the last 24h (or last week) for each of my projects and right now I have to click through every project to get that info.

Then, if you want to drill down in more detail you can open the detail page of a project.

🙏🏽 🙏🏽 🙏🏽

**Comment by metmarkosaric** [comment]
**Update:** We’ve made some improvements to the “My Sites” page:

* You can now search for any of your sites
* There’s a graph for an overview of the traffic 
* And a comparison to the previous period so you can see the trend

Have fun exploring!

**Reply by toniengelhardt** [reply]
Amazing! This is really cool, thanks a lot 🙏🏽

**Comment by kuttnerd** [comment]
one quick question: is it possible to add the graphs in the overview of the self-hosted version, too? We use 2.0.0 but couldn't find anything on this in the documentation. 
Many thx for plausible!
